wiki/Exchange-Markets-By-Country.md

```diff
@@ -18,7 +18,6 @@ The ccxt library currently supports the following cryptocurrency exchange market
 | Brazil                   | [![flowbtc](https://user-images.githubusercontent.com/51840849/87443317-01c0d080-c5fe-11ea-95c2-9ebe1a8fafd9.jpg)](https://one.ndax.io/bfQiSL)                                                    | flowbtc            | [flowBTC](https://one.ndax.io/bfQiSL)                                                    | [![API Version *](https://img.shields.io/badge/*-lightgray)](https://www.flowbtc.com.br/api.html)                                                  |
 | Brazil                   | [![mercado](https://user-images.githubusercontent.com/1294454/27837060-e7c58714-60ea-11e7-9192-f05e86adb83f.jpg)](https://www.mercadobitcoin.com.br)                                              | mercado            | [Mercado Bitcoin](https://www.mercadobitcoin.com.br)                                     | [![API Version 3](https://img.shields.io/badge/3-lightgray)](https://www.mercadobitcoin.com.br/api-doc)                                            |
 | Brazil                   | [![novadax](https://user-images.githubusercontent.com/1294454/92337550-2b085500-f0b3-11ea-98e7-5794fb07dd3b.jpg)](https://www.novadax.com.br/?s=ccxt)                                             | novadax            | [NovaDAX](https://www.novadax.com.br/?s=ccxt)                                            | [![API Version 1](https://img.shields.io/badge/1-lightgray)](https://doc.novadax.com/pt-BR/)                                                       |
-| Brazil                   | [![ripio](https://user-images.githubusercontent.com/1294454/94507548-a83d6a80-0218-11eb-9998-28b9cec54165.jpg)](https://exchange.ripio.com)                                                       | ripio              | [Ripio](https://exchange.ripio.com)                                                      | [![API Version 1](https://img.shields.io/badge/1-lightgray)](https://exchange.ripio.com/en/api/)                                                   |
```

Comment:
I wouldn't remove Brazil. Maybe comment is as: Ripio / BitcoinTrade

Reply:
@sserrano44 adding a comment to the table breaks it, as you can see in the image bellow. When we migrate BitcoinTrade to RipioTrade in Brazil we'll need to create a PR to CCXT anyway, to add Brazil as a supported country in the code, in that PR we can also add the line to the .md file again. Could we do it like that?

<img width="1281" alt="Captura de Tela 2022-07-21 às 13 46 28" src="https://user-images.githubusercontent.com/1892491/180270045-f17e3565-fa65-47ad-8060-0cb8d7776e5e.png">

Reply:
ok